docs/security-considerations.rst

```diff
@@ -259,7 +259,7 @@ more special edge cases for signed numbers.
 
 Try to use ``require`` to limit the size of inputs to a reasonable range and use the
 :ref:`SMT checker<smt_checker>` to find potential overflows, or use a library like
-`SafeMath <https://github.com/OpenZeppelin/openzeppelin-solidity/blob/master/contracts/math/SafeMath.sol>`_
+`SafeMath <https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/math/SafeMath.sol>`_
```

Comment:
Since the old link is valid, why the update?

Reply:
https://github.com/OpenZeppelin/openzeppelin-solidity redirects to https://github.com/OpenZeppelin/openzeppelin-contracts

I think they must have renamed it and left the old one only to keep the old links working. I think the more up to date one has a better chance of surviving in the long term.